examples/command_line/swift_c_module/BUILD

```diff
@@ -0,0 +1,27 @@
+load("@build_bazel_rules_swift//swift:swift.bzl", "swift_c_module")
+
+cc_library(
+    name = "lib_impl",
```

Comment:
To keep the project looking nice, can we rename this to `c_lib` and remove the below `alias`? Will prevent the name clashing, and another example already shows the `alias` case.